autoload/gutentags.vim

```diff
@@ -98,7 +98,7 @@ function! gutentags#setup_gutentags() abort
     " Try and find what tags file we should manage.
     call gutentags#trace("Scanning buffer '" . bufname('%') . "' for gutentags setup...")
     try
-        let b:gutentags_root = gutentags#get_project_root(expand('%:h'))
+        let b:gutentags_root = gutentags#get_project_root(expand('%:p:h'))
```

Comment:
`expand()` should always be called with `1` as the second argument, to avoid `'wildignore'` pain.